30. Как в ES6 обрабатывать исключения с помощью блоков `try...catch`?
В ES6 блок `try...catch` позволяет обрабатывать исключения. Можно использовать `catch` без указания имени ошибки